src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -383,36 +383,22 @@
         "fallbackVariableName": "presetTestsDefault"
       }
     },
-    "renderer": {
-      "displayName": "Renderer",
-      "description": "Controls the renderer used for iOS, Android, Catalyst and WebAssembly targets",
+    "nativeRendering": {
+      "displayName": "Native rendering",
+      "description": "Use native rendering on targets platforms",
```

Comment:
```suggestion
      "description": "Use the native API-based renderers",
```

Reply:
It was agreed to use "Use native rendering on all target platforms".